README.md

```diff
@@ -463,19 +463,31 @@ require("mobile-cli-lib").devicesService.mapAbstractToTcpPort("4df18f307d8a8f1b"
 	});
 ```
 
-* `getApplicationsAvailableForDebugging(deviceIdentifier: string): Promise<string[]>` - This function checks the proc/net/unix file of the device for web views connected to abstract ports and returns the applications identifiers.
+* `getDebuggableApps(deviceIdentifier: string): Promise<Mobile.IAndroidApplicationInformation[]>` - This function checks the proc/net/unix file of the device for web views connected to abstract ports and returns information about the applications.
```

Comment:
Previously, this method's return type was `Promise<string[]>`, now it's changed to `Promise<Mobile.IAndroidApplicationInformation[]>`, however it's not entirely clear just from reading this example what `Mobile.IAndroidApplicationInformation` actually provides.
A description of the interface is needed IMO